Formula/libav.rb

```diff
@@ -60,11 +60,9 @@ class Libav < Formula
   depends_on "speex" => :optional
   depends_on "theora" => :optional
 
-  # Fixes the use of a removed identifier in libvpx;
-  # will be fixed in the next release.
   patch do
-    url "https://github.com/libav/libav/commit/4d05e9392f84702e3c833efa86e84c7f1cf5f612.patch"
-    sha256 "78f02e231f3931a6630ec4293994fc6933c6a1c3d1dd501989155236843c47f9"
+    url "https://bugzilla.libav.org/attachment.cgi?id=656"
```

Comment:
Probably add https://bugzilla.libav.org/show_bug.cgi?id=1033 in a comment for easier progress tracking?

By the way, https://bugzilla.libav.org/robots.txt blocks search engines. So crazy.